longbox_scrapers/adapters/atomic_avenue.py

```
@@ -880,3 +891,270 @@ def _extract_issue_id_from_url(self, url: str) -> str | None:
+        try:
+            async with self._http as http:
+                for letter in letters:
+                    params = {"indexLetter": letter}
+                    url = f"{self.BASE_URL}/atomic/Publishers.aspx?{urlencode(params)}"
+
+                    try:
+                        response = await http.get(url)
+                        response.raise_for_status()
+                        letter_publishers = self._parse_publishers_page(response.text)
+                        publishers.extend(letter_publishers)
+                    except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)